File: src/main/java/com/tonitealive/server/ToniteAliveApplication.java
package com.tonitealive.server;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.EnableAspectJAutoProxy;

@SpringBootApplication
@EnableAspectJAutoProxy
public class ToniteAliveApplication {

    public static void main(String[] args) {
        SpringApplication.run(ToniteAliveApplication.class, args);
    }

}

File: src/main/kotlin/com/tonitealive/server/ToniteAliveApplication.kt
package com.tonitealive.server

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication
open class ToniteAliveApplication {

    companion object {
        @JvmStatic fun main(args: Array<String>) {
            SpringApplication.run(ToniteAliveApplication::class.java, *args)
        }
    }

}